lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -287,6 +299,9 @@ class DetailsRenderer {
         const strValue = String(value);
         return this._renderText(strValue);
       }
+      case 'markdowntext': {
```

Comment:
`markdown` seems pretty unambiguous to me wdyt

Reply:
done